client/my-sites/checkout/checkout-thank-you/jetpack-checkout-siteless-thank-you-completed.tsx

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import React, { FC } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+import { Card } from '@automattic/components';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'calypso/components/main';
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import {
+	isProductsListFetching as getIsProductListFetching,
+	getProductName,
+} from 'calypso/state/products-list/selectors';
+import JetpackLogo from 'calypso/components/jetpack-logo';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface Props {
+	productSlug: string | 'no_product';
+}
+
+const JetpackCheckoutSitelessThankYouCompleted: FC< Props > = ( { productSlug } ) => {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const hasProductInfo = productSlug !== 'no_product';
+
+	const productName = useSelector( ( state ) =>
+		hasProductInfo ? getProductName( state, productSlug ) : null
+	) as string | null;
+
+	const isProductListFetching = useSelector( ( state ) =>
+		getIsProductListFetching( state )
+	) as boolean;
```

Comment:
Similarly here, you should be able to add jsdoc to [this function](https://github.com/Automattic/wp-calypso/blob/e3de25f37b98f73da1121cd6cb4ea26948836277/client/state/products-list/selectors/is-products-list-fetching.js#L6) to make it clear that it returns a `{boolean}`, which will not only help you but also anyone else who needs it in the future.